release/release_test.go

```diff
@@ -41,10 +41,10 @@ func (s *ReleaseTestSuite) TestSetup(c *C) {
 	c.Check(release.Series, Equals, "16")
 }
 
-func makeMockLSBRelease(c *C) string {
+func makeMockOSRelease(c *C) string {
```

Comment:
Sorry, nevermind. Please ignore all of that. The file is trivially small, so all irrelevant.